src/Reviews/ReviewModel.cs

```diff
@@ -127,5 +128,16 @@ public class ReviewModel
         /// Gets or sets the review workflow associated with the review.
         /// </summary>
         public ReviewWorkflowModel ReviewWorkflow { get; set; }
+
+        /// <summary>
+        /// Gets or sets the state via a string name.
+        /// </summary>
+        [JsonIgnore]
+        public string StateName
+        {
+            get => this.State.ToString();
+
+            set => this.State = value.ToEnum<ReviewStateType>();
+        }
```

Comment:
I dont think we need this. State serializes fine to string and if not, the enum should have the Json string serializer attribute added. This seems redundant. If the UI uses this property, let's remove it, and instead use State.